packages/@expo/fingerprint/package.json

```
@@ -44,9 +44,11 @@
   "homepage": "https://github.com/expo/expo/tree/main/packages/@expo/fingerprint#readme",
   "dependencies": {
     "@expo/spawn-async": "^1.7.2",
+    "arg": "4.1.0",
```

```suggestion
    "arg": "^5.0.2",
```

nit: align the version with @expo/cli that to reduce multiple `arg` installed in a blank project (without expo-updates installed)